hashes/src/internal_macros.rs

```diff
@@ -207,6 +206,24 @@ macro_rules! hash_type {
             fn internal_new(arr: [u8; $bits / 8]) -> Self { Hash(arr) }
 
             fn internal_engine() -> HashEngine { Default::default() }
+
+            /// Construts this wrapper type from an array.
```

Comment:
Typo `Construts` here and other places as well.

This comment seems like it was cut'n paste but doesn't quite fit, the `Hash` is not a wrapper type, right?

Reply:
It can be considered a wrapper type around an array but I do not like that perception (even though it's being promoted by `Deref`/`Index` traits) so I'm happy to fix that.